packages/dart/noports_core/lib/src/sshnp/util/srvd_channel/notification_request_message.dart

```diff
@@ -1,12 +1,29 @@
 import 'dart:convert';
 
+import 'package:noports_core/sshnp_foundation.dart';
+
 class SocketRendezvousRequestMessage {
-  late String sessionId;
-  late String atSignA;
-  late String atSignB;
-  late bool authenticateSocketA;
-  late bool authenticateSocketB;
-  late String clientNonce;
+  final String sessionId;
+  final String atSignA;
+  final String atSignB;
+  final bool authenticateSocketA;
+  final bool authenticateSocketB;
+  final String clientNonce;
+  final RelayAuthMode relayAuthMode;
+  final String? relayAuthAesKey;
+  final bool only443;
```

Comment:
Three new params - relayAuthMode, relayAuthAesKey, and a flag "only443" (probably want to improve on this)